suite-native/formatters/src/components/CryptoAmountFormatter.tsx

```diff
@@ -33,13 +34,15 @@ export const CryptoAmountFormatter = ({
     const maxDisplayedDecimals = decimals ?? networks[network].decimals;
 
     const stringValue = G.isNumber(value) ? value.toString() : value;
-    const formattedValue = formatter.format(stringValue, {
+
+    const localizedValue = localizeNumber(stringValue);
```

Comment:
Can you please test how fast this function is? This component needs to remain fastest as possible otherwise our lists will be slow. You can use simple `performance.now()`

Reply:
+- 0.7 ms is the mean average for the whole localization and formatting 
localizeNumber takes +- 90% of this time

Reply:
if I dont use locale and just force the thousands commas, I can make the whole function finish in +- 0.07 ms

reusing parts of `localizeNumber` to make this:

```
export const formatNumberWithThousandCommas = (
    value: number | string | BigNumber,
    minDecimals = 0,
    maxDecimals?: number,
) => {
    if (maxDecimals !== undefined && maxDecimals < minDecimals) {
        throw Error(
            `maxDecimals (${maxDecimals}) cannot be lower than minDecimals (${minDecimals})`,
        );
    }

    const amount = new BigNumber(value);

    const getDecimalsLength = () => {
        const originalDecimalsLegth = amount.decimalPlaces() ?? 0;
        if (originalDecimalsLegth < minDecimals) {
            return minDecimals;
        }
        if (maxDecimals !== undefined && originalDecimalsLegth > maxDecimals) {
            // Remove trailing zeroes after formatting:
            return new BigNumber(amount.toFixed(maxDecimals)).decimalPlaces() ?? maxDecimals;
        }
        return originalDecimalsLegth;
    };

    return amount.toFormat(getDecimalsLength(), {
        decimalSeparator: '.',
        groupSize: 3,
        groupSeparator: ',',
    });
};
```

Reply:
If it's under ms it seems reasonable

Reply:
10x speed up seems nice, please try to do that

Reply:
bffbd782e